App/index.js

```diff
@@ -1,6 +1,43 @@
 import React from 'react';
-import { HomeScreen } from './screens/HomeScreen';
+import { ApolloProvider } from 'react-apollo';
+import { createStackNavigator, createAppContainer } from 'react-navigation';
+import client from './lib/apollo-client';
+import HomeScreen from './screens/HomeScreen';
+import ProfileScreen from './screens/ProfileScreen';
 
-export default function App() {
-  return <HomeScreen />;
-}
+const AppNavigator = createStackNavigator(
+  {
+    Home: HomeScreen,
+    Profile: ProfileScreen,
+  },
+  {
+    initialRouteName: 'Home',
+    defaultNavigationOptions: {
+      headerStyle: {
+        backgroundColor: '#006064',
```

Comment:
@ssanusi It is a good practice to use external styling.

Reply:
thanks for the heads up i have implemented the feedback